test/nsidc-smap/smap_utils.py

```diff
@@ -91,7 +91,7 @@ def _generate_reference_files(test_config: dict, in_dir: Path, out_dir: Path):
     input_files = [
         Path("SPL2SMA_subset_bounding_box.tif"),
         Path("SPL2SMA_subset_by_geojson.h5"),
-        Path("SPL2SMP_E_2_subset_by_variable.tif"),
+        Path("SPL3SMA_subset_by_variable.tif"),
```

Comment:
This is a helper function that I use to generate reference files, I fixed that name elsewhere but didn't change it here.